.github/workflows/ios-ci.yml

```diff
@@ -164,6 +164,45 @@ jobs:
             ${{ env.ios_cpp_test_artifacts_dir }}/CppUnitTests.xctest.zip
             ${{ env.ios_cpp_test_artifacts_dir }}/CppUnitTestsApp.ipa
 
+      # Size test (Bloaty)
+
+      - name: Build dynamic library for size test (Bloaty)
+        if: matrix.renderer == 'drawable'
+        run: |
+          bazel build //platform/ios:MapLibre.dynamic --//:renderer=metal --//:maplibre_platform=ios --compilation_mode="opt" --copt -g --strip never --output_groups=+dsyms --apple_generate_dsym
```

Comment:
Build with `-Oz` here to prevent the inliner behavior from distorting the optimal size configuration. Otherwise things may get aggressively inlined which distorts the metric we're interested in.

Additionally, if we can, build only for armv8/AArch64. By default I think the xcframework should have both binaries in it (x86-64 and armv8). Bloaty gets confused by that and tends to pick one, usually x86-64 in my testing. Arm binaries are going to be bigger just by virtue of the instruction set, so make sure we're comparing arm.

Reply:
Yes it contains both:

```
$ ls MapLibre.xcframework
Info.plist*                  ios-arm64/                   ios-arm64_x86_64-simulator/
```

But I am extracting the armv8 dynamic library from the XCFramework:

```
cp MapLibre.xcframework/ios-arm64/MapLibre.framework/MapLibre MapLibre_dynamic
```

```
 $ file MapLibre_dynamic
MapLibre_dynamic: Mach-O 64-bit dynamically linked shared library arm64
```

So should be fine.